software/util/sdoowl.py

```diff
@@ -23,9 +23,9 @@
 from rdflib.compare import graph_diff
 from rdflib.namespace import RDFS, RDF
 
-from xml.etree import ElementTree as ET
+from xml.etree import ElementTree
 from xml.dom import minidom
-from xml.etree.ElementTree import Element, SubElement, Comment, tostring
+from xml.etree.ElementTree import Element, SubElement
```

Comment:
This line seems not used now, as all refs are fully qualified wrt to `ElementTree`.